content/millennium/r4/diagnostic/observation.md

```diff
@@ -119,6 +119,37 @@ Notes:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Retrieve by id
+
+List an individual Observation by its id:
+
+    GET /Observation/M-id
```

Comment:
shouldn't this just be:
    GET /Observation/:id

Reply:
For observations we will have the prefix

Reply:
is the ID we return on a resource only the number? this indicates that we should return "M-" plus what the ID returns on the resource.

Binary is similar https://fhir.cerner.com/millennium/r4/other/binary/#retrieve-by-id

Reply:
Fixed it as a part of this PR https://github.com/cerner/fhir.cerner.com/pull/445